src/App.tsx

```diff
@@ -1,26 +1,47 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from './app/hooks';
+import { todosSlice } from './features/todos';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [listLoading, setListLoading] = useState(false);
+  const dispatch = useDispatch();
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  useEffect(() => {
+    setListLoading(true);
+    getTodos()
+      .then(array => {
+        dispatch(todosSlice.actions.setTodos(array));
+      })
+      .finally(() => setListLoading(false));
+  }, []);
```

Comment:
to avoid lint error it's better just add `dispatch` to dependencies list (it's memorized too, so actually it will work the same), or at leat disable lint for this row